[WARNING] No exception occurred in online testing but java.lang.AssertionError occurred now.
[WARNING] Ignoring different exception occurrence in offline test.
[WARNING] The remaining test execution may produce different results.
[ERROR] java.lang.AssertionError: assertion failed
[ERROR] 	at scala.Predef$.assert(Predef.scala)
[ERROR] 	at modbat.dsl.Model$.assert(Model.scala)
[ERROR] 	at modbat.dsl.Model$class.assert(Model.scala)
[ERROR] 	at modbat.test.TestObserver6.assert(TestObserver6.scala)
[ERROR] 	at modbat.test.TestObserver6$$anonfun$1.apply$mcV$sp(TestObserver6.scala)
[ERROR] 	at modbat.test.TestObserver6$$anonfun$1.apply(TestObserver6.scala)
[ERROR] 	at modbat.test.TestObserver6$$anonfun$1.apply(TestObserver6.scala)
[WARNING] java.lang.AssertionError: assertion failed occurred, aborting.
[ERROR] java.lang.AssertionError: assertion failed
[ERROR] 	at scala.Predef$.assert(Predef.scala)
[ERROR] 	at modbat.dsl.Model$.assert(Model.scala)
[ERROR] 	at modbat.dsl.Model$class.assert(Model.scala)
[ERROR] 	at modbat.test.TestObserver6.assert(TestObserver6.scala)
[ERROR] 	at modbat.test.TestObserver6$$anonfun$1.apply$mcV$sp(TestObserver6.scala)
[ERROR] 	at modbat.test.TestObserver6$$anonfun$1.apply(TestObserver6.scala)
[ERROR] 	at modbat.test.TestObserver6$$anonfun$1.apply(TestObserver6.scala)
[WARNING] java.lang.AssertionError: assertion failed occurred, aborting.
[ERROR] java.lang.AssertionError: assertion failed
[ERROR] 	at scala.Predef$.assert(Predef.scala)
[ERROR] 	at modbat.dsl.Model$.assert(Model.scala)
[ERROR] 	at modbat.dsl.Model$class.assert(Model.scala)
[ERROR] 	at modbat.test.TestObserver6.assert(TestObserver6.scala)
[ERROR] 	at modbat.test.TestObserver6$$anonfun$3.apply$mcV$sp(TestObserver6.scala)
[ERROR] 	at modbat.test.TestObserver6$$anonfun$3.apply(TestObserver6.scala)
[ERROR] 	at modbat.test.TestObserver6$$anonfun$3.apply(TestObserver6.scala)
